assets/tests/testHelpers/wrappers.tsx

```diff
@@ -0,0 +1,24 @@
+import { useReducer } from "react"
+import React from "react"
+import { reducer, State } from "../../src/state"
+import stateFactory from "../factories/applicationState"
+import { StateDispatchProvider } from "../../src/contexts/stateDispatchContext"
+
+export const RealDispatchWrapper = ({
+  children,
+  initialState,
+}: {
+  children: JSX.Element
+  initialState: State
+}) => {
+  const [state, dispatch] = useReducer(
+    reducer,
+    undefined,
+    () => initialState || stateFactory.build()
+  )
+  return (
+    <StateDispatchProvider state={state} dispatch={dispatch}>
+      {children}
+    </StateDispatchProvider>
+  )
+}
```

Comment:
Still iffy on this but tests that interact with the global state needed this to reduce boilerplate and complexity in tests.

As a behavioral testing tool, it's very useful.